develop/api_metadata.rst

```diff
@@ -110,6 +110,40 @@ The shardstorage column in pg_dist_shard indicates the type of storage used for
 |                |                      | | distributed file_fdw tables)                                              |
 +----------------+----------------------+-----------------------------------------------------------------------------+
 
+.. _citus_shards:
+
+Shard information view
+~~~~~~~~~~~~~~~~~~~~~~
+
+In addition to the low-level shard metadata table described above, Citus provides a ``citus_shards`` view to easily check:
+
+* Where each shard is (node, and port),
```

Comment:
there is also the colocation id in the output, not sure if it is worth adding here.

Reply:
Ah you already explained it below :)